spec/features/team/view_team_members_spec.rb

```diff
@@ -35,14 +35,14 @@
 
     context 'when there are many users in my organisation' do
       let!(:user_1) { create(:user, email: 'bob@example.gov.uk', organisation: organisation) }
-      let!(:user_2) { create(:user, email: 'amada@example.gov.uk', organisation: organisation) }
-      let!(:user_3) { create(:user, email: 'zara@example.gov.uk', organisation: organisation) }
+      let!(:user_2) { create(:user, name: 'amada', organisation: organisation) }
+      let!(:user_3) { create(:user, name: 'zara', organisation: organisation) }
 
       it 'renders all team members within my organisation in alphabetical order' do
         sign_in_user user
         visit team_members_path
 
-        expect(page.body).to match(/#{user_2.email}.*#{user_1.email}.*#{user.email}.*#{user_3.email}/m)
+        expect(page.body).to match(/#{user_2.name}.*#{user_1.email}.*#{user.email}.*#{user_3.name}/m)
```

Comment:
I don't think this spec covers the feature, which is "sort by username, or email if username is missing". This spec only sorts by name, as `user_1` has an auto generated name of "bob" (https://github.com/alphagov/govwifi-admin/blob/master/spec/factories/users.rb#L7). 

For full coverage you would need to ensure at least one of the users in the list here doesn't have a name. You could just set it to an empty string in the declaration.